server/app/controllers/applicant/ApplicantProgramBlocksController.java

```diff
@@ -833,14 +824,6 @@ private CompletionStage<Result> renderErrorOrRedirectToRequestedPage(
       return supplyAsync(() -> redirect(applicantRoutes.review(profile, applicantId, programId)));
     }
     if (applicantRequestedAction == ApplicantRequestedAction.PREVIOUS_BLOCK) {
-      if (onAddressCorrectionPage) {
-        // When an applicant is on the address correction view and clicks "Previous", we want them
-        // to go back to the block with the address question. Address correction isn't a defined
-        // block in the program definition, so `blockId` represents the block with the address
-        // question and we just need to go back to that block.
-        return getBlockPage(profile, applicantId, programId, blockId, inReview, flashingMap);
-      }
-
```

Comment:
Wanted to give some detail on why this was removed.

Earlier, we had thought that the address correction page would still have the three classic buttons of "Review", "Previous", and "Save and next". If you click "Previous" while on the address correction page, we wanted to take the user back to the block **with** the address question, not the block **before** the address question. This if block and the `onAddressCorrectionPage` boolean was being used to handle that.

Now that we've removed the "Previous" button from the address correction page, we don't need this custom logic to handle that case. (And note that if the `SAVE_ON_ALL_ACTIONS` flag is off, we still handle this correctly in https://github.com/civiform/civiform/blob/e87bdabb0a49eabdc042edfc314b04f894adfa86/server/app/views/applicant/AddressCorrectionBlockView.java#L265-L267.)